### PR TITLE
add RaftEntryTooLarge error type

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
@disksing @siddontang PTAL

### CODE REVIEW

**Comment by disksing** [comment]
LGTM

**Comment by zhangjinpeng1987** [comment]
@BusyJay PTAL

File: proto/errorpb.proto [hunk 1]
```diff
@@ -37,13 +37,19 @@ message ServerIsBusy {
 message StaleCommand {
 }
 
+message RaftEntryTooLarge {
+    optional uint64 region_id  = 1;
+    optional uint64 entry_size = 2;
```
Comment (siddontang) [inline proto/errorpb.proto]: Do we need to add the max_size here ?

Comment (zhangjinpeng1987) [inline proto/errorpb.proto]: NO need.

**Comment by zhangjinpeng1987** [comment]
PING

**Comment by siddontang** [comment]
LGTM

**Pull request merged**